website/blog/2019-07-15-complete-table-scan.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Complete Table Scan: A Quantitative Assessment
+author: Orri Erling
+authorURL: https://www.linkedin.com/in/orrierling/
+authorFBID: 100026224749124
+---
+
+In the previous article we looked at the abstract problem statement and possibilities inherent in scanning tables. In this piece we look at the quantitative upside with Presto. We look at a number of queries and explain the findings.
+
+The initial impulse motivating this work is the observation that table scan is by far the #1 operator in Presto workloads I have seen. This is a little over half of all Presto CPU, with repartitioning a distant second, at around 1/10 of the total. The other half of the motivation is ready opportunity: Presto in its pre-Aria state does almost none of the things that are common in table scan.
+
+<!--truncate-->
```

Comment:
What does this do?

Reply:
The blog has an index as well as individual posts. In the index it posts a snippet of each blog. This tag determines where the snippet ends.